internal/reader/home_chain.go

```diff
@@ -36,6 +41,17 @@ type HomeChain interface {
 	GetFChain() (map[cciptypes.ChainSelector]int, error)
 	// GetOCRConfigs Gets the OCR3Configs for a given donID and pluginType
 	GetOCRConfigs(ctx context.Context, donID uint32, pluginType uint8) ([]OCR3ConfigWithMeta, error)
+
+	// RMNHome specific methods
+	// GetRMNNodesInfo gets the RMNHomeNodeInfo for the given configDigest
+	GetRMNNodesInfo(configDigest cciptypes.Bytes32) ([]rmn.RMNHomeNodeInfo, error)
```

Comment:
Not sure I like the RMN import here, I feel like it will lead to import cycles at some point. Maybe move `RMNHomeNodeInfo` to a `types` subpackage within `rmn` to reduce the likelihood?